srcpkgs/hobbits/template

```diff
@@ -1,28 +1,50 @@
 # Template file for 'hobbits'
 pkgname=hobbits
-version=0.52.0
+version=0.53.1
 revision=1
+_pffft_commit=233c26659185ff55958e96e4c25f3fcafe249a65
+_pffft_dir="jpommier-pffft-${_pffft_commit:0:12}"
+create_wrksrc=yes
+build_wrksrc="hobbits-${version}"
 build_style=cmake
+configure_args="-DPFFFT_ROOT=../${_pffft_dir}"
 hostmakedepends="pkg-config qt5-host-tools qt5-qmake python3"
-makedepends="fftw-devel libusb-devel libpcap-devel python3-devel qt5-devel"
+makedepends="libusb-devel libpcap-devel python3-devel qt5-devel"
 short_desc="Multi-platform GUI for bit-based analysis, processing and visualization"
 maintainer="classabbyamp <dev@kb6.ee>"
-license="MIT, GPL-2.0-or-later"
+license="MIT, custom:FFTPACK"
 homepage="https://mahlet-inc.github.io"
 changelog="https://raw.githubusercontent.com/Mahlet-Inc/hobbits/master/CHANGELOG.md"
-distfiles="https://github.com/Mahlet-Inc/hobbits/archive/refs/tags/v${version}.tar.gz"
-checksum=3332e02380ab46649f0ec2066b63b6e0fb04b55cdca973d28880f1607ebbf27e
+distfiles="https://github.com/Mahlet-Inc/hobbits/archive/refs/tags/v${version}.tar.gz
+ https://bitbucket.org/jpommier/pffft/get/${_pffft_commit}.tar.gz"
+checksum="e63c51228c3f00f56a2ebf79fec1975544b1c6c986d58dff69643da46148316b
+ @c1df1ae9d807f2e0fc2391bd8e714ac5bfc76a54b20ef3de3d82acb026c79f74"
 
 case "$XBPS_TARGET_MACHINE" in
 	arm*|aarch64*|ppc*) CXXFLAGS+=" -fsigned-char"
 esac
 
+case "$XBPS_TARGET_MACHINE" in
+	arm*) broken="requires NEON instructions" ;;
+	aarch64*) PFFFT_CFLAGS="" ;;
+	ppc*) PFFFT_CFLAGS="-maltivec -mabi=altivec" ;;
+	*) PFFFT_CFLAGS="-msse -mfpmath=sse" ;;
```

Comment:
We should probably only enable sse on i686 and x86_64, not on all other architectures.